### ISSUE TITLE
unable to reorder array elements in 1.5.24 when using "create" views

### ISSUE THREAD

**Issue by DookTibs** [gitana/alpaca#633]
When running Alpaca with a "create" view like "web-create-horizontal", reordering array elements doesn't work properly for me. The animation swap runs but after it completes, all array elements get blanked out.

It works correctly when using an "edit" view instead. Tested on Chrome and Safari.

I did not test this on older versions of Alpaca.

Fiddle illustrating the problem:
https://jsfiddle.net/DookTibs/w9wmcLja/

**Comment by seanvree** [comment]
yeah, this is a bug in 1.5.24.  This is NOT an issue on older versions.
